test/integration/patients/worklist.js

```diff
@@ -377,6 +377,11 @@ context('worklist page', function() {
     cy
       .url()
       .should('contain', 'patient-action/2/form/1');
+
+    cy
+      .visit('/worklist/powned-by-me')
```

Comment:
@luketlancaster this 2nd visit in a specify clears all coverage before it.

Reply:
oh dang. I was wondering if it needed to be in another `specify` guess so